devtools/platform-descriptor-json-plugin/src/main/java/io/quarkus/maven/GenerateExtensionsJsonMojo.java

```diff
@@ -9,20 +9,10 @@
 import java.nio.file.FileSystem;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-import javax.json.Json;
-import javax.json.JsonArray;
-import javax.json.JsonArrayBuilder;
-import javax.json.JsonObject;
-import javax.json.JsonObjectBuilder;
-import javax.json.JsonReader;
-import javax.json.JsonValue;
-import javax.json.JsonWriter;
-import javax.json.JsonWriterFactory;
+import java.util.*;
```

Comment:
We don't use star imports, can you use the appropriate code formatter?

Reply:
the build passes so I assume I am already using the appropriate code formatter.

Reply:
oh damn - is the PR's i made on having devtools share the formatting still not in master. ? gddmmit

Reply:
fixed manually